### ISSUE TITLE
Getting gradle error when I clone and try to build an APK

### ISSUE THREAD

**Issue by tomascordero** [rampage128/cardroid#7]
Error message:
Build model 'org.jetbrains.plugins.gradle.model.BuildScriptClasspathModel' for root project 'cardroid'	33ms

Steps to recreate:
1. Clone repo into new folder
2. open project in android studio
3. run "build APK(s)"

This happens with a fresh install of android studio. Is there any steps I might be missing?

**Comment by rampage128** [comment]
Hi, sorry I can not reproduce your Issue.

I just got a new computer (windows) and didn't install anything yet, so I took this issue as a reason to set up my dev environment.

I just:
* Installed Android Studio
* Installed git for windows
* Cloned the project
* Opened the project in Android Studio
* Got the message to install platform-28 from the SDK manager
* Installed platform-28
* Hit the sync-button
* Hit the build-button

Project syncs and builds successfully...

Even updating gradle to 5.6.4 (as prompted by AS) does not cause any issue.

Which gradle version are you using?
